aunt-leahs-app/src/pages/VolunteerCheckInPage.jsx

```diff
@@ -56,18 +66,41 @@ function VolunteerCheckInPage({ location, volunteer }) {
 			</div>
 			<br />
 			<div className='check-in-custom-button'>
-				<CustomButton size="small" color="primary"
-					onClick={postShift}>
+				<CustomButton size="small" color="primary" onClick={postShift} isDisabled={isFormSubmitted || isSubmissionSuccessful}>
 					Submit
 				</CustomButton>
 			</div>
+			{isFormSubmitted ?
+				<SubmissionStatus isFormSubmitted isSubmissionSuccessful className='check-in-submission-status' /> :
+				null
+			}
```

Comment:
Use an if block instead

Reply:
https://react-cn.github.io/react/tips/if-else-in-JSX.html

I can't use an if block here